lib/Net/SAML2/Protocol/AuthnRequest.pm

```diff
@@ -147,6 +151,20 @@ has 'RequestedAuthnContext_Comparison' => (
     default => 'exact'
 );
 
+has 'force_authn' => (
+    isa     => 'Str',
```

Comment:
These appear to be booleans, so why don't we use `isa => 'Bool'` here?

Reply:
I did think about that.  Will revise